psydac/api/equation.py

```diff
@@ -14,7 +14,7 @@
 from psydac.fem.basic                import FemField
 from psydac.linalg.solvers           import inverse
 
-__all__ = ('DiscreteEquation',)
+__all__ = ('l2_boundary_projection', 'DiscreteEquation',)
```

Comment:
@yguclu I will replace all such instances of round parentheses with bracket parentheses tomorrow.

Reply:
750ffe893abcb82fcbc8c358ab4128932c29fc3b

Reply:
Why do you prefer a list to a tuple?

Reply:
Yes please. `__all__` should be a tuple.